frontend-site/src/modules/staticData.js

```diff
@@ -53,6 +54,12 @@ export const servicesOffered = Object.freeze([
     description: 'Need a break a stressful week of classes? Pick up a board game, located in the LUG office, and grab some friends. There are plenty of games to choose from.',
     url: '',
   },
+  {
+    name: 'Minecraft',
+    image: require("@/assets/minecrat.jpg"),
```

Comment:
Spelling: `minecrat.png` -> `minecraft.png`

Also, I don't see this asset in the PR at the time of this comment.

Reply:
Oh yea I forgot to comment that out. I was going to upload a photo from google but I dont want to upload a copywritted image so I was just going to wait.